Tasks/PerformActionOnMachineGroup/task.json

```diff
@@ -16,7 +16,15 @@
         "Patch": 6
     },
     "minimumAgentVersion": "1.83.0",
-    "inputs": [      
+    "inputs": [ 
+        {
+            "name": "ConnectedServiceName",
+            "type": "connectedService:Azure",
+            "label": "Azure Subscription",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Azure subscription under which the Resource Group is deployed."
```

Comment:
Subscription, and agree with Sravya, text should say so - optional for pre-existing machine groups.